### DISCUSSION TITLE
Add 2.0-3.0 checkmark to change studios back to extension version

### DISCUSSION THREAD

**Discussion by ItsEnderium** [ScratchAddons/ScratchAddons#2866] [Ideas]
Here's what the new studios look like:
![image](https://user-images.githubusercontent.com/82263560/123208472-0f8dff80-d4b7-11eb-96ef-cd0bb3093e60.png)
And here's what the Scratch Addons version looks like:
![Screenshot 2021-06-24 06 41 44](https://user-images.githubusercontent.com/82263560/123208614-5aa81280-d4b7-11eb-8c31-578bc828d8d0.png)
You should add the ability to change it back to the Scratch Addons version in 2.0-3.0 as a checkmark, as I prefer it.
![Screenshot 2021-06-24 06 44 53](https://user-images.githubusercontent.com/82263560/123208873-b83c5f00-d4b7-11eb-8159-c480d022dc9a.png)
We still need to add three dots for each person on the list for managers, as we can't remove features.
![image](https://user-images.githubusercontent.com/82263560/123208948-d73af100-d4b7-11eb-8d6c-171efd963c9b.png)

**Comment by apple502j** [comment]
I am not sure if this is possible. Marking as "likely to be rejected but unsure".

**Reply by mxmou** [reply]
If it just changes the appearance, it's possible, but it probably won't be part of Scratch 2.0 → 3.0.
